man/systemd.mount.xml

```diff
@@ -194,6 +194,22 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>x-systemd.before=</option></term>
```

Comment:
Any chance you can add x-systemd.after= at the same time? Yes, you don't need it for your usecase, but it would be easy to ad this at the same time, and document at the same time.

(no need to add another varlistentry to the man page for the before thing, just document it in the same one)